skit.API/Extensions/IdentityExtension.cs

```diff
@@ -31,7 +31,7 @@ public static IServiceCollection AddIdentityConfig(this IServiceCollection servi
             options.Password.RequireNonAlphanumeric = true;
 
             options.User.RequireUniqueEmail = true;
-            options.SignIn.RequireConfirmedEmail = false; //TODO do zmiany na true jak będą maile
+            options.SignIn.RequireConfirmedEmail = false;
```

Comment:
Tutaj nie powinno być `true`? Jak nie to bym wrócił z komentarzem.

Reply:
Nie bo wtedy jak utworzysz konto to się nie zalogujesz dopóki maila nie potwierdzisz, a ustaliliśmy że ma przenosić od razu do panelu edycji

Reply:
Ma to sens